core/src/main/scala/scalaz/FingerTree.scala

```diff
@@ -807,18 +805,6 @@ sealed abstract class FingerTree[V, A](implicit measurer: Reducer[A, V]) {
     )
   }
 
-  private def traverseFinger[F[_], A, B, V2](digit: Finger[V, A])(f: A => F[B])(implicit ms: Reducer[B, V2], F: Applicative[F]): F[Finger[V2, B]] = {
```

Comment:
Ah, my second favorite reason for using `private` all over the place.  (My first is that incremental compiles are faster 🏎 )